howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,85 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services. If you want to serve the appliance over HTTPS using a valid SSL/TLS certificate, this document will walk you through the steps in generating and installing a valid SSL/TLS certificate on Anbox Cloud Appliance.
+
+[note type="information" status="Note"]This document assumes you have Anbox Cloud Appliance installed. If you haven't, please follow the [instruction](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
+
+## Add a DNS Record
+
+Setting up DNS redirection depends on your DNS provider. Refer to the documentation of your provider to create a DNS record pointing to the IP/DNS of the AWS instance where the Anbox Cloud appliance is running on.
+
+## Configure the location
+
+Configure the location for the appliance using the created DNS name.
+
+    sudo snap set anbox-cloud-appliance experimental.location=<your DNS name>
+
+[note type="information" status="Note"]This option is experimental and will be removed in the future release when a better replacement exists.[/note]
```

Comment:
```suggestion
[note type="information" status="Note"]This option is experimental. It will be removed in a future release when a replacement exists.[/note]
```